__tests__/integration/one-app.spec.js

```diff
@@ -1668,3 +1705,323 @@ describe('heapdump', () => {
     expect(size).toBeLessThanOrEqual(100 * oneMegabyte);
   });
 });
+
+describe('Routes sanity checks', () => {
```

Comment:
these should be moved to be in the first block https://github.com/americanexpress/one-app/blob/main/__tests__/integration/one-app.spec.js#L59